modules/ListView.js

```diff
@@ -233,10 +250,10 @@
 
       this.collection = collection;
 
-      this.listenTo(this.collection, 'remove', removeChildView);
-      this.listenTo(this.collection, 'add', addChildView);
-      this.listenTo(this.collection, 'sort', this.reorder);
-      this.listenTo(this.collection, 'reset', this.update);
+      this.listenTo(this.collection, 'remove', removeItemView, this);
```

Comment:
I don't think you need the last this param.  That should already bind to this by default (since it is the same this that listenTo is called from).